google/cloud/aiplatform/tensorboard/tensorboard_resource.py

```diff
@@ -782,3 +785,331 @@ def list(
+            tensorboard_id (str):
+                Optional. The resource ID of the Tensorboard to create the TensorboardTimeSeries in.
+            tensorboard_experiment_id (str):
+                Optional. The ID of the TensorboardExperiment to create the TensorboardTimeSeries in.
+            value_type (Union[gca_tensorboard_time_series.TensorboardTimeSeries.ValueType, str]):
+                Optional. Type of TensorboardTimeSeries value. One of 'SCALAR', 'TENSOR', 'BLOB_SEQUENCE'.
+            display_name (str):
+                Optional. User provided name of this
+                TensorboardTimeSeries. This value should be
+                unique among all TensorboardTimeSeries resources
+                belonging to the same TensorboardRun resource
+                (parent resource).
+
+                If not provided tensorboard_time_series_id will be used.
+            description (str):
+                Optional. Description of this TensorboardTimeseries.
+            project (str):
+                Optional. Project to upload this model to. Overrides project set in
+                aiplatform.init.
+            location (str):
+                Optional. Location to upload this model to. Overrides location set in
+                aiplatform.init.
+            credentials (auth_credentials.Credentials):
+                Optional. Custom credentials to use to upload this model. Overrides
+                credentials set in aiplatform.init.
+            request_metadata (Sequence[Tuple[str, str]]):
+                Optional. Strings which should be sent along with the request as metadata.
+        Returns:
+            TensorboardTimeSeries: The TensorboardTimeSeries resource.
+        """
+
+        if display_name:
+            utils.validate_display_name(display_name)
+
+        display_name = display_name or tensorboard_time_series_id
+
+        if isinstance(value_type, str):
+            value_type = getattr(
+                gca_tensorboard_time_series.TensorboardTimeSeries.ValueType, value_type
+            )
+
+        api_client = cls._instantiate_client(location=location, credentials=credentials)
+
+        parent = utils.full_resource_name(
+            resource_name=tensorboard_run_name,
+            resource_noun=TensorboardRun._resource_noun,
+            parse_resource_name_method=TensorboardRun._parse_resource_name,
+            format_resource_name_method=TensorboardRun._format_resource_name,
+            parent_resource_name_fields={
+                Tensorboard._resource_noun: tensorboard_id,
+                TensorboardExperiment._resource_noun: tensorboard_experiment_id,
+            },
+            project=project,
+            location=location,
+        )
+
+        gapic_tensorboard_time_series = gca_tensorboard_time_series.TensorboardTimeSeries(
+            display_name=display_name, description=description, value_type=value_type,
```

Comment:
Could you pass in plugin_name as well like https://github.com/googleapis/python-aiplatform/blob/d72a254e97cf74f3fdd55a32a4af86737243593a/google/cloud/aiplatform/tensorboard/uploader.py#L438 ?

The scalar plugin name: https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/scalar/metadata.py#L21

Reply:
... and if you plan to write non-scalar data you might need plugin_content as well, which I don't have much knowledge on how to generate.

Reply:
Added